### PR TITLE
docs: Fix grammatical & typo errors in the README.md file.

### PR DESCRIPTION
**Pull request by roubalsehgal**
## Proposed changes
Resolved some typo & grammatical errors in the README file.

## Types of changes

- [x] Documentation Update (README file updated)

## Checklist

- [x] I have read the **CONTRIBUTION** guidelines.

### CODE REVIEW

**Comment by roubalsehgal** [comment]
Hello reviewers,

I was just going through the README of this repo & just spotted these minor errors so thought of giving it a PR to make it better. Please let me know if it is a valid PR or not.

If not please share your feedback!

File: README.md [hunk 1]
```diff
@@ -78,4 +78,4 @@ Read more on our website: [https://www.postman.com/](https://www.postman.com/)
 
 ---
 
-If you have issues or code contribution pertaining to Postman legacy version, head over to the [postmanlabs/postman-chrome-extension-legacy](https://github.com/postmanlabs/postman-chrome-extension-legacy) repository.
+If you have issues or code contributions about the Postman legacy version, head over to the [postmanlabs/postman-chrome-extension-legacy](https://github.com/postmanlabs/postman-chrome-extension-legacy) repository.
```
Comment (arlemi) [inline README.md]: Can we keep pertaining here?

**Review by arlemi — CHANGES_REQUESTED** [review]
Only added one little comment, everything else looks good, thanks for having a look at it!

File: README.md (hunk 1 above)
Comment (roubalsehgal) [reply README.md]: Yes, sure! 😀

**Review by arlemi — APPROVED** [review]
Looks good, thanks a lot for the updates! 🥳

**Pull request merged**